pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/EventTriggerDao.java

```diff
@@ -66,4 +66,12 @@ default long insertUserRegisteredTrigger() {
     default long insertExitRequestTrigger() {
         return getJdbiEventTrigger().insert(EventTriggerType.EXIT_REQUEST);
     }
+
+    default long insertChildContactTrigger() {
+        return getJdbiEventTrigger().insert(EventTriggerType.CHILD_CONTACT);
```

Comment:
Why is `child_contact` an event trigger type??

Reply:
hmm. looks I got confused with one of the event triggers. cleaned it up. thank you